pallets/gear/src/lib.rs

```diff
@@ -1500,6 +1503,9 @@ pub mod pallet {
                 ),
             );
 
+            // MIKITA different gas for 2 branchs
```

Comment:
it's like there 3 different cases:
- going to queue (active program)
- going to waitlist (uninitialised program)
- going to events (user interactions)